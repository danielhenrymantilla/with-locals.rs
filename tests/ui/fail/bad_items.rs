include!("../prelude.rs");

#[with]
use self;

#[with]
const _: () = {};

const _: () = {
    #[with]
    static __: () = {};
};

const _: () = {
    #[with]
    type __ = ();
};

const _: () = {
    #[with]
    struct __ {}
};

const _: () = {
    #[with]
    enum __ {}
};

const _: () = {
    #[with]
    union __ {}
};
