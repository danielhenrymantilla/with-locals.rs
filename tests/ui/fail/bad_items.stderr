error: expected `fn`
  --> $DIR/bad_items.rs:31:5
   |
31 |     union __ {}
   |     ^^^^^

error: expected `fn`
  --> $DIR/bad_items.rs:26:5
   |
26 |     enum __ {}
   |     ^^^^

error: expected `fn`
  --> $DIR/bad_items.rs:21:5
   |
21 |     struct __ {}
   |     ^^^^^^

error: expected `fn`
  --> $DIR/bad_items.rs:16:5
   |
16 |     type __ = ();
   |     ^^^^

error: expected `fn`
  --> $DIR/bad_items.rs:11:5
   |
11 |     static __: () = {};
   |     ^^^^^^

error: expected `fn`
 --> $DIR/bad_items.rs:7:7
  |
7 | const _: () = {};
  |       ^

error: expected `fn`
 --> $DIR/bad_items.rs:4:1
  |
4 | use self;
  | ^^^
